andela/casablanca-cfh
# #156096163 Ft gaming screen ui 156096163

#### What does this PR do?
 It redesigns the game screen page

#### Description of Task to be completed?
Re-Design UI for game screen

#### How should this be manually tested?
- clone the repo and change into the project directory
- run `npm install` to install required packages
- create a `.env` file and add all the required environment variables as specified in `.env.example`
- use `npm run start:dev` to start up the server
- visit `http://localhost:3000/#!/app` to see and test the game page

#### Any background context you want to provide?
N/A

### Screenshots
<img width="1280" alt="screen shot 2018-04-10 at 6 46 24 pm" src="https://user-images.githubusercontent.com/32877552/38574059-c0c73090-3cef-11e8-8039-739e561af748.png">
<img width="1280" alt="screen shot 2018-04-10 at 6 46 46 pm" src="https://user-images.githubusercontent.com/32877552/38574061-c176da2c-3cef-11e8-87bb-efb6068c7a8f.png">
<img width="1280" alt="screen shot 2018-04-10 at 6 47 07 pm" src="https://user-images.githubusercontent.com/32877552/38574065-c21f3172-3cef-11e8-8927-5a85e884c6db.png">
<img width="1280" alt="screen shot 2018-04-10 at 6 47 14 pm" src="https://user-images.githubusercontent.com/32877552/38574066-c248a868-3cef-11e8-9f98-d869b0996d20.png">

#### What are the relevant pivotal tracker stories?(if applicable)
#156096163
